packages/_private/e2e-demo-test/src/test.page.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @copyright
+ * Copyright(c) 2020 SICK AG
+ */
+
+import { Locator, Page, Response } from '@playwright/test';
+import locators from './test.selector';
+
+/**
+ * Gauge visual regression test class
+ */
+export default class TestPage {
+  /**
+   * constructor
+   *
+   * @param page - _page
+   * @param _browser - _browser
+   * @param _scriptName - _scriptName
+   */
+  constructor(private page: Page) {
```

Comment:
Just curious: Is this done via auto-binding? I do see the references to `this.page` in the code, but it is not set in the constructor or anywhere else.